### DISCUSSION TITLE
a way to DM other developers

### DISCUSSION THREAD

**Discussion by Ishaan282** [community/community#75862] [New to GitHub]
### Select Topic Area

Question

### Body

is their any way to have chat with other developers on github 
this will be helpful when contributing in open-source projects

**Comment by albbus-stack** [comment]
You could open up a thread on the discussions section of the repo if you are talking about public contributions stuff. Otherwise suggest the repo owner to create a Discord server in which you can talk publicly & privately.

**Reply by Ishaan282** [reply]
in open source projects it's kinda hard 
recently i have solved a problem about a bug 
i tried contacting the developer through comments who helped me but it seems they didn't got any notification so i though it would be nice if GitHub could add this simple feature

**Reply by albbus-stack** [reply]
just ping them directly if you want them to get the notification anyway

**Comment by arimartorano** [comment]
No, that's not an option but would be a really cool future-feature to have.

**Comment by grandmas-favorite** [comment]
Welcome to the GitHub Community, @Ishaan1106, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
